x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -199,6 +199,11 @@ export const allowedExperimentalValues = Object.freeze({
    */
   analyzerDatePickersAndSourcererDisabled: false,
 
+  /**
+   * Enables graph visualization in alerts flyout
+   */
+  graphVisualizationInFlyoutEnabled: false,
```

Comment:
open for better naming suggestions

Reply:
I attached new videos to the PR's description, you can see now alert's preview and hosts having graph preview